solutions/utilities.py

```diff
@@ -0,0 +1,7 @@
+
+def input_reader(day: int):
+    return_list = []
+    with open(f"input_day{day}.txt", "r") as f:
```

Comment:
This is fine, but only works if the input file is in the directory where the command is invoked.